larray/core/array.py

```diff
@@ -7168,11 +7221,13 @@ def zeros_like(array, title='', dtype=None, order='K'):
     """
     if not title:
         title = array.title
-    return LArray(np.zeros_like(array, dtype, order), array.axes, title)
+    if meta is None:
```

Comment:
I don't think this is a good idea to copy metadata by default in all the *_like functions. At least it does not match current usage. Usually, our users create new arrays to represent a different "concept" than the original array. They just want the same axes. Having an easy option to copy the metadata might be a good idea (e.g. meta='copy'). Unsure it worth it compared to the user doing it explicitly:
```python
>>> a = ndtest((2, 3))
>>> b = zeros_like(a, meta=a.meta)
```
In either case, it would be nice if you mentioned in the docstring whether or not metadata is copied.